lib/http_clients/creditas/person_deletion_api.ex

```diff
@@ -1,7 +1,16 @@
 defmodule HttpClients.Creditas.PersonDeletionApi do
   @moduledoc false
 
-  alias HttpClients.Creditas.PersonDeletionApi.PersonDeletion
+  alias HttpClients.Creditas.PersonDeletionApi.{Acknowledgment, PersonDeletion}
+
+  @spec acknowledgments(Tesla.Client.t(), Acknowledgment.t()) :: {:error, any} | {:ok}
+  def acknowledgments(client, ack) do
+    case Tesla.post(client, "/#{ack.person_deletion_id}/acknowledgments", build_ack_payload(ack)) do
+      {:ok, %Tesla.Env{status: 200}} -> {:ok}
```

Comment:
```suggestion
      {:ok, %Tesla.Env{status: 200} = response} -> {:ok, response}
```
uma tupla de 1 elemento não é usual, essa resposta é fixa será?

Reply:
combinamos de retornar um `:ok` pq o retorno é o próprio corpo da requisição